browser_tests/fixtures/ws.ts

```diff
@@ -29,7 +29,7 @@ export const webSocketFixture = base.extend<{
             function ([data, url]) {
               if (!url) {
                 // If no URL specified, use page URL
-                const u = new URL(window.location.toString())
+                const u = new URL(window.location.toString().split('#')[0])
```

Comment:
Could you use this? https://developer.mozilla.org/en-US/docs/Web/API/Location/hash